mantidproject/mantid
# Indirect - MSDFit Interface - Add Plot Guess feature

Currently in the MSDFit interface, there is no Plot Guess feature (as seen in the other Indirect Data Analysis Fit Interfaces).

This PR adds the plot guess feature to the MSDFit interface.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > MSDFit
2. Supply the provided file in the Input field.
3. For each model in the Model Input drop-down menu:
    1. Select the model in the Model Input drop-down menu.
    2. Check the Plot Guess checkbox.
    3. Ensure the guess is plotted (green plot) in the top preview plot.
    4. Move the blue fit selectors in the top preview plot and ensure the guess plot is updated.

**Must be merged after #20980**

**Test Files:**
[msdfit_test_file.zip](https://github.com/mantidproject/mantid/files/1402311/msdfit_test_file.zip)



Fixes #20975 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
